packages/nimble-components/src/combobox/index.ts

```diff
@@ -682,6 +683,18 @@ export class Combobox
         }
     }
 
+    /**
+     * Need to update even when options is empty.
+     * @internal
+     * @remarks
```

Comment:
I might remark that this is essentially the same implementation as what was in FAST (w/o the guard).